R-packages/covidcast/R/plot.R

```diff
@@ -383,54 +390,39 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
       is_hawaii = substr(hrr_name, 1, 2) == 'HI',
       is_pr = substr(hrr_name, 1, 2) == 'PR',
       color = ifelse(hrr_num %in% geo, col_fun(val[hrr_num]), missing_col))
-
-    main_df = shift_main(map_df)
-    hawaii_df = shift_hawaii(map_df)
-    alaska_df = shift_alaska(map_df, alaska_centroid)
-    pr_df = shift_pr(map_df, pr_centroid)
-
-    main_col = main_df$color
-    hawaii_col = hawaii_df$color
-    alaska_col = alaska_df$color
-    pr_col = pr_df$color
-  }
-
-  # Create the polygon layer 
-  if (attributes(x)$metadata$geo_type == "county" || 
-      attributes(x)$metadata$geo_type == "state") {
-    aes = ggplot2::aes
-    geom_args = list()
-    geom_args$color = border_col
-    geom_args$size = border_size
-    geom_args$mapping = aes(x = x, y = y, group = group)
-    geom_args$fill = map_col
-    geom_args$data = map_df
-    polygon_layer = do.call(ggplot2::geom_polygon, geom_args)
-    coord_layer = ggplot2::coord_equal()
-  }
-  else if (attributes(x)$metadata$geo_type == "msa" || 
-          attributes(x)$metadata$geo_type == "hrr") {
-    aes = ggplot2::aes
-    geom_args = list()
-    geom_args$color = border_col
-    geom_args$size = border_size
-    geom_args$mapping = aes(geometry=geometry)
-    coord_args = list()
-
-    geom_args$fill = main_col
-    geom_args$data = main_df
-    main_layer = do.call(ggplot2::geom_sf, geom_args)
-    geom_args$fill = pr_col
-    geom_args$data = pr_df
-    pr_layer = do.call(ggplot2::geom_sf, geom_args)
-    geom_args$fill = hawaii_col
-    geom_args$data = hawaii_df
-    hawaii_layer = do.call(ggplot2::geom_sf, geom_args)
-    geom_args$fill = alaska_col
-    geom_args$data = alaska_df
-    alaska_layer = do.call(ggplot2::geom_sf, geom_args)
-    coord_layer = do.call(ggplot2::coord_sf, coord_args)
   }
+
+  main_df = shift_main(map_df)
+  hawaii_df = shift_hawaii(map_df)
+  alaska_df = shift_alaska(map_df)
+  pr_df = shift_pr(map_df)
+
+  main_col = main_df$color
+  hawaii_col = hawaii_df$color
+  alaska_col = alaska_df$color
+  pr_col = pr_df$color
+
+  # Create the polygon layers
+  aes = ggplot2::aes
```

Comment:
Can get rid of this too

Reply:
It's referenced a lot and the code is less messy this way, I'd rather leave it.